src/namer/namer.ts

```diff
@@ -79,7 +79,6 @@ async function process(session: Session<CodeModel>) {
   // pascal-case and capitzalize acronym operation groups and their operations
   for (const group of values(model.operationGroups)) {
     const details = <Language>group.language.go;
-    const opGroupName = capitalizeAcronyms(pascalCase(group.$key));
```

Comment:
Why dont we want to have this naming fix anymore?

Reply:
It's dead code, but you bring up a good point that maybe it shouldn't be.  Let me take a look.

Reply:
The reference to it was removed in https://github.com/Azure/autorest.go/pull/313.  Apparently I forgot to remove the definition then.  :(